src/main/java/net/sf/jabref/shared/security/Password.java

```diff
@@ -0,0 +1,68 @@
+package net.sf.jabref.shared.security;
+
+import java.security.GeneralSecurityException;
+import java.util.Base64;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.SecretKeySpec;
+
+/**
+ *   {@link Password} contains methods which are useful to encrypt and decrypt passwords using symetric algorithms.
+ */
+public class Password {
+
+    private static final String ALGORITHM = "AES";
+    private static final String STATIC_KEY = "ThisIsA128bitKey";
+    private final byte[] key;
+    private final String phrase;
+
+
+    /**
+     * @param phrase Phrase which should be encrypted or decrypted
+     * @param key Secret key which is used to improve symmetric encryption
+     */
+    public Password(String phrase, String key) {
+        this.phrase = phrase;
+        this.key = convertToAESKey(key);
+    }
+
+    /**
+     *  Encrypts the set phrase/password with strong symmetric encryption algorithms.
+     *
```

Comment:
Well, AES 128 bit is a joke and not really secure, I personally would prefer at least 256biz, but using stronger encryption is only possible with the Cryptographic Extensions Policy and unfortunately  those fall under the US import/export regulation laws. 

It would be possible to check if they are installed and use a greater key size. Not sure it if is really necessary here.